FirebaseDatabase/Sources/Core/FPersistentConnection.m

```diff
@@ -707,6 +726,39 @@ - (void)sendPut:(NSNumber *)index {
             }];
 }
 
+- (void)sendGet:(NSNumber *)index {
```

Comment:
Note: In general, API names in Objective-C should include the name of the first argument and be somewhat readable in their own (something like `sendGetAtIndex`). I am not necessarily pushing for this here (especially since this matches `sendPut`) , but I just wanted to raise this in this PR as it applies in a couple of places.

See https://g3doc.corp.google.com/company/teams/objc-readability/style_guide.md?cl=head#objective-c-method-names